test/fixtures/3.0/ping/DEVELOPING.md

```diff
@@ -0,0 +1,36 @@
+# Developer's Guide
```

Comment:
I find it super confusing to have a Developer's Guide that's applicable only to a small part of the project. Please remove this file entirely, or move it top-level and make necessary edits to make the content meaningful to connector developers.